src/pages/auth/reset-device/[id].vue

```diff
@@ -29,69 +41,28 @@ meta:
           required
         />
       </div>
-
-      <AppNotification
-        v-if="hasError && !errorCode"
-        variant="error"
-        class="mb-4"
-        :title="t('resetDevice.errorTitle')"
-      >
-        <p>
-          <i18n-t keypath="resetDevice.errorText">
-            <template #newLink>
-              <router-link to="/auth/lost-device" class="underline">{{
-                t('resetDevice.errorLink')
-              }}</router-link>
-            </template>
-          </i18n-t>
-        </p>
-      </AppNotification>
-
-      <AppNotification
-        v-if="hasError && errorCode"
-        variant="error"
-        class="mb-4"
-        :title="t('resetDevice.errorTitle')"
-      >
-        <p>{{ t('resetDevice.errors.' + errorCode) }}</p>
-      </AppNotification>
-
-      <AppButton
-        variant="link"
-        :disabled="validation.$invalid || loading"
-        class="mb-4 w-full"
-        type="submit"
-        @click="handleSubmit"
-        >{{ t('resetDevice.deleteDevice') }}</AppButton
-      >
-    </form>
+    </AppForm>
   </AuthBox>
 </template>
 
 <script lang="ts" setup>
-import { reactive, ref } from 'vue';
+import { reactive } from 'vue';
 import { useRoute, useRouter } from 'vue-router';
 import { useI18n } from 'vue-i18n';
-import useVuelidate from '@vuelidate/core';
 
 import AppInput from '@components/forms/AppInput.vue';
-import AppButton from '@components/button/AppButton.vue';
 import AppTitle from '@components/AppTitle.vue';
 import AuthBox from '@components/AuthBox.vue';
-import AppNotification from '@components/AppNotification.vue';
 
 import { resetDeviceComplete } from '@utils/api/reset-security-flow';
 import { updateCurrentUser } from '@store/index';
 import { isInternalUrl } from '@utils/index';
-import { isRequestError } from '@utils/api';
 
-import { addNotification } from '@store/notifications';
+import AppForm from '@components/forms/AppForm.vue';
```

Comment:
If we want to keep the style I started, then this import should be below the import of the other components